events/class_tests/event_tests.py

```diff
@@ -41,9 +41,8 @@ def test_delete_event(self, persist_event):
         assert persist_event not in Event.objects.all()
 
     def test_exist_event(self):
-        assert Event.objects.get(title='event1')
-        assert Event.objects.get(title='event2')
-        assert Event.objects.get(title='event3')
+        for title_id in ['1', '2', '3']:
```

Comment:
you don't have to add the surrounded apostrophes 
```suggestion
        for title_id in [1, 2, 3]:
        or 
        for title_id in range(1, 4):
        
```

Reply:
done. thanks 😄